rust/connlib/shared/src/messages/client.rs

```diff
@@ -85,6 +87,7 @@ impl ResourceDescriptionInternet {
         crate::callbacks::ResourceDescriptionInternet {
             id: self.id,
             sites: self.sites,
+            disableable: false,
```

Comment:
```suggestion
            disableable: true,
```

Reply:
I think it's better if we set it as `false` here until we add it into the portal's API?